_config/features_comparison.yml

```diff
@@ -0,0 +1,241 @@
+features:
+  - name: "ACID"
+    jta: true
+    jts: true
+    xts: true
+    comp: false
+    rts: true
+    stm: true
+    blacktie: true
+    arjuna: true
+    txoj: true
+  - name: "Synchronizations"
+    jta: true
+    jts: true
+    xts: true
+    comp: false
+    rts: true
+    stm: false
+    blacktie: false
+    arjuna: true
+    txoj: false
+  - name: "Nested transactions"
+    jta: true
+    jts: true
+    xts: true
+    comp: false
+    rts: false
+    stm: true
+    blacktie: false
+    arjuna: true
+    txoj: true
+  - name: "Automated failure recovery"
+    jta: true
+    jts: true
+    xts: true
+    comp: false
+    rts: true
+    stm: false
+    blacktie: true
+    arjuna: true
+    txoj: true
+  - name: "Transaction log (file system, JDBC, HornetQ)"
+    jta: true
+    jts: true
+    xts: true
+    comp: false
+    rts: true
+    stm: true
+    blacktie: true
+    arjuna: true
+    txoj: true
+  - name: "JTA 1.2 API"
+    jta: true
+    jts: true
+    xts: true
+    comp: false
+    rts: false
```

Comment:
You can enlist XAResource, because bridge creates JTA transaction in the server side.